### ISSUE TITLE
[Feature Request] Multi-line item support

### ISSUE THREAD

**Issue by buu700** [mattboehm/dottask#1]
It'd be useful if typing shift+enter while editing an item added a line break.

**Comment by mattboehm** [comment]
Agreed, Thanks! I've thought this many times and keep forgetting that I haven't implemented it yet. I'll make this a priority and sorry for not noticing this ticket sooner.
